Description: Avoid tripping over the concurrent import in compileall
 Minimal includes compileall but not concurrent.
Forwarded: not-needed
--- a/Lib/compileall.py
+++ b/Lib/compileall.py
@@ -85,10 +85,10 @@
         raise ValueError('workers must be greater or equal to 0')
     if workers != 1:
         # Check if this is a system where ProcessPoolExecutor can function.
-        from concurrent.futures.process import _check_system_limits
         try:
+            from concurrent.futures.process import _check_system_limits
             _check_system_limits()
-        except NotImplementedError:
+        except (NotImplementedError, ImportError):
             workers = 1
         else:
             from concurrent.futures import ProcessPoolExecutor
